stock-db/stock.py

```diff
@@ -0,0 +1,67 @@
+import sqlite3
+import pandas as pd
+conn = sqlite3.connect("stocks.sqlite")
+cur = conn.cursor()
+
+table_query = cur.execute("SELECT name FROM sqlite_master WHERE type='table'")
+tables = table_query.fetchall()
+
+msft_query = cur.execute("""
+        SELECT * 
+        FROM STOCK_DATA
+        WHERE Symbol = 'MSFT';
+""")
+msftdata = msft_query.fetchall()
+msft = pd.DataFrame(msftdata)
+
+cur.execute("CREATE TABLE msft (`index`, Date, Open, High, Low, Close, Volume, Adj Close, Symbol)")
+
+cur.executemany("""
+            INSERT INTO msft
+            VALUES(?,?,?,?,?,?,?,?,?)
+""",msft.values)
+
+aapl_query = cur.execute("""
+        SELECT * 
+        FROM STOCK_DATA
+        WHERE Symbol = 'AAPL';
+""")
+aapldata = aapl_query.fetchall()
+aapl = pd.DataFrame(aapldata)
+
+cur.execute("CREATE TABLE aapl (`index`, Date, Open, High, Low, Close, Volume, Adj Close, Symbol)")
+
+cur.executemany("""
+            INSERT INTO aapl
+            VALUES(?,?,?,?,?,?,?,?,?)
+""",aapl.values)
+
+msft_date_query = cur.execute("""
+        SELECT MAX(Date), MIN(Date)
+        FROM msft
+""")
+msftdatedata = msft_date_query.fetchall()
+msft_date = pd.DataFrame(msftdatedata)
```

Comment:
Don't forget column names (hint: can also use pd.read_sql() to automatically get column names). This also applies to dataframes below.